tests/features/fixtures/mazerunner/src/main/java/com/bugsnag/android/mazerunner/scenarios/CXXConfigurationMetadataNativeCrashScenario.java

```diff
@@ -0,0 +1,37 @@
+package com.bugsnag.android.mazerunner.scenarios;
+
+import com.bugsnag.android.Bugsnag;
+import com.bugsnag.android.Configuration;
+
+import android.content.Context;
+
+import androidx.annotation.NonNull;
+
+public class CXXConfigurationMetadataNativeCrashScenario extends Scenario {
```

Comment:
There doesn't seem to be corresponding scenarios for any of the `features/fixtures` test suite. If we want to make a decision to stop writing tests for that suite and focus on the browser tests instead I'd be ok with that, as long as it's documented somewhere in the repo/confluence.